src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -31,6 +32,8 @@ public class CommandTestUtil {
     public static final String VALID_NAME_BOB = "Bob Choo";
     public static final String VALID_PHONE_AMY = "11111111";
     public static final String VALID_PHONE_BOB = "22222222";
+    public static final String VALID_NOK_PHONE_AMY = "99998877";
+    public static final String VALID_NOK_PHONE_BOB = "98341256";
```

Comment:
Since a valid Nok Phone "only contains number and should be at least 3 digits long", I think it would be better if we add more variants to the sample Nok Phone instead of fixing it to be 8 digit long.

Reply:
the at least 3 digits long is actually in AB3 previously.
should we stick to the convention of fixing it 8 digits long and that the starting number can only be 8 or 9 ?

Reply:
I feel like we should not restrict ourselves in the Singapore context with a fixed 8-digit long phone number and could be more flexible. But I'm open to the idea of having a fixed 8-digit idea.

Besides, if we allow more complex numbers, I was thinking about whether we should allow space, "+", and "-" for the phone number as well. Since sometimes people may put a space between numbers like "1111 1111" or in some countries where the numbers are long, people may put a dash in between like "133-3333-3333". The same applies to the "+" for the country code like "+65". If we want to allow for more complex phone numbers for different countries, these cases should be considered.

Reply:
These are good points! Perhaps we can discuss this during the meeting and open another issue to address this. For now, this PR will just add next of kin phone to the fields to set up the base structure.